src/main/java/no/unit/nva/cristin/projects/Constants.java

```diff
@@ -22,4 +22,5 @@
     public static final String CRISTIN_LANGUAGE_PARAM = "lang";
     public static final String TITLE = "title";
     public static final String LANGUAGE = "language";
+    public static final String QUESTION_MARK = "?";
```

Comment:
PARAMETER_ARRAY_PREFIX. If it is to name it question mark, then better to use the actual question mark.

Reply:
I looked in the URI spec, it is actually called "question mark"

Reply:
My bad  then :)